test/loopback-dl.test.js

```diff
@@ -1560,35 +1563,35 @@ describe('DataAccessObject', function() {
     (function() { model._normalize({filter: {x: undefined}}); }).should.throw(/`undefined` in query/);
   });
 
-  it('should skip GeoPoint', function() {
+  it('does not coerce GeoPoint', function() {
     where = model._coerce({location: {near: {lng: 10, lat: 20}, maxDistance: 20}});
     assert.deepEqual(where, {location: {near: {lng: 10, lat: 20}, maxDistance: 20}});
   });
 
-  it('should skip null values', function() {
+  it('does not coerce null values', function() {
     where = model._coerce({date: null});
     assert.deepEqual(where, {date: null});
   });
 
-  it('should skip undefined values', function() {
+  it('does not coerce undefined values', function() {
     where = model._coerce({date: undefined});
     assert.deepEqual(where, {date: undefined});
   });
 
-  it('should skip conversion if a simple property produces NaN for numbers',
+  it('does not coerce to a number a simple value that produces NaN',
     function() {
       where = model._coerce({age: 'xyz'});
       assert.deepEqual(where, {age: 'xyz'});
     });
 
-  it('should skip conversion if an array property produces NaN for numbers',
+  it('does not coerce to a number a simple value in an array that produces NaN',
```

Comment:
does not coerce value to a number for a simple value in an array that produces NaN